Assets/HotmaxEditorVR/inputManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class inputManager : MonoBehaviour
+{
+    // -- controller refs
+    public static Valve.VR.InteractionSystem.Hand hand1;
+    public static Valve.VR.InteractionSystem.Hand hand2;
+    public static SteamVR_TrackedController trackedController1;
+    public static SteamVR_TrackedController trackedController2;
+    private SteamVR_TrackedObject trackedObject2;
+    public static SteamVR_Controller.Device device2;
+
+
+    private stateManager _stateManagerMutatorRef;
+    private bool _selectedObjectIsActive = stateManager.selectedObjectIsActive;
+    private stateManager.editorModes _editorMode = stateManager.editorMode;
+    
+
+
+    private void Awake()
+    {
+        //Match up the hardware index with the "steamVR Hand2"
+        //So the Hand2 is always the starting rightmost hand reletive to the HMD
+        //Not sure why steamVR's select hand option doesn't do this, let me know if you figure it out
+        hand1 = GameObject.Find("Hand1").GetComponent<Valve.VR.InteractionSystem.Hand>();
+        hand2 = GameObject.Find("Hand2").GetComponent<Valve.VR.InteractionSystem.Hand>();
+        trackedController1 = hand1.gameObject.GetComponent<SteamVR_TrackedController>();
+        trackedController2 = hand2.gameObject.GetComponent<SteamVR_TrackedController>();
+        trackedObject2 = hand2.GetComponent<SteamVR_TrackedObject>();
+        device2 = SteamVR_Controller.Input((int)trackedObject2.index);
+
+
+        Invoke("setControllerIndecies", 1.2f);
```

Comment:
whats the deal with invoke?

Reply:
It is waiting for a steamVR method which fires 1s after the app starts.  The purpose of the steamVR method is to find the hardware indices of the controllers.